client/my-sites/themes/themes-magic-search-card/index.jsx

```diff
@@ -89,6 +156,16 @@ const ThemesMagicSearchCard = React.createClass( {
 		);
 	},
 
+	renderWelcomeSign: function() {
+		const taxonomies = Object.keys( this.state.taxonomies );
+
+		return (
+			taxonomies.map( ( taxonomy, i ) => {
+				return <span className="suggestions__taxonomy" key={ i }>{ taxonomy }</span>;
```

Comment:
Nitpick -- since `taxonomy` is unique here, you could use it for `key`, and drop `i`.